### PR TITLE
completed read me for frontend. Additional info maybe added in the fu…

### PR DESCRIPTION
**Pull request by Developer3027**
…ture.

# Description
Read Me complete

Fixes # (issue)
No buges or fixes required

## Type of change
information / documentation

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] Test A - reviewed by me on Google and Vivaldi

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,133 +38,92 @@ Project under review and scored by [Code Climate](https://codeclimate.com/github
...
+- React does not need to refresh the entire page.
 
-#### Front end deployed to `🚫insert service here`
+#### Front end deployed to https://aws.amazon.com
 
-#### [Back end](🚫link to back end repo here) built using:
+#### [Back end](https://github.com/Lambda-School-Labs/ping-catcher-be) built using:
 
-#### 🚫 back end framework goes here
+#### Node, Express, PostgressQl
```
Comment (debauchery1st) [inline README.md:76]: PostgreSQL, also known as Postgres

File: README.md [hunk 2]
```diff
@@ -42,133 +38,92 @@ Project under review and scored by [Code Climate](https://codeclimate.com/github
...
-🚫Replace text below with a description of the API
-
-Volunteering your time; it pays you and your whole community fantastic dividends. Maybe there's a happy little waterfall happening over here. You can spend all day playing with mountains. We don't have to be committed. We are just playing here. You have freedom here. The only guide is your heart. It's cold, but it's beautiful.
+In order for the app to function correctly, the user must set up their own environment variables. There should be a .env file containing the following:
 
-# 3️⃣ Environment Variables
+Production Okta server
 
-In order for the app to function correctly, the user must set up their own environment variables. There should be a .env file containing the following:
+- REACT_APP_OKTA_CLIENT_ID=0oahbvrj0hNcoZBXv4x6
```
Comment (debauchery1st) [inline README.md:94]: remove secret.
describe the key.

File: README.md [hunk 3]
```diff
@@ -42,133 +38,92 @@ Project under review and scored by [Code Climate](https://codeclimate.com/github
...
+- REACT_APP_OKTA_ORG_URL=https://dev-605205.okta.com
```
Comment (debauchery1st) [inline README.md:95]: "

File: README.md [hunk 4]
```diff
@@ -42,133 +38,92 @@ Project under review and scored by [Code Climate](https://codeclimate.com/github
...
+- SPA_CLIENT_ID=0oahbvrj0hNcoZBXv4x6
```
Comment (debauchery1st) [inline README.md:96]: "

File: README.md [hunk 5]
```diff
@@ -42,133 +38,92 @@ Project under review and scored by [Code Climate](https://codeclimate.com/github
...
-    *  REACT_APP_authDomain - when you set up your Firebase project, this information will be in the dashboard
-    *  REACT_APP_databaseURL - in the Firebase dashboard
-    *  REACT_APP_projectID - in the Firebase dashboard
-    *  REACT_APP_storageBucket - in the Firebase dashboard
-    *  REACT_APP_messagingSenderId - in the Firebase dashboard
-    *  REACT_APP_stripe_API - this is your public Stripe API key, generated in the Stripe dashboard
-    *  REACT_APP_backendURL - optional for your local development server
-    *  REACT_APP_clientid - this is the Stripe_connect clientID, generated in Stripe_connect settings
-    *  REACT_APP_stripe_plan - this is the ID for a second Stripe subscription plan, generated under Stripe products
+- REACT_APP_OKTA_IMPLICIT_CALLBACK_URL=http://localhost:8080/implicit/callback
```
Comment (debauchery1st) [inline README.md:100]: "

File: README.md [hunk 6]
```diff
@@ -42,133 +38,92 @@ Project under review and scored by [Code Climate](https://codeclimate.com/github
...
+- REACT_APP_OKTA_API_MESSAGES=http://localhost:8080/api/messages
```
Comment (debauchery1st) [inline README.md:101]: "

File: README.md [hunk 7]
```diff
@@ -42,133 +38,92 @@ Project under review and scored by [Code Climate](https://codeclimate.com/github
...
+- PORT=8080
```
Comment (debauchery1st) [inline README.md:102]: "

File: README.md (hunk 1 above)
Comment (Developer3027) [reply README.md]: corrected on new push

File: README.md (hunk 2 above)
Comment (Developer3027) [reply README.md]: corrected on new push

File: README.md (hunk 3 above)
Comment (Developer3027) [reply README.md]: corrected on new push

File: README.md (hunk 4 above)
Comment (Developer3027) [reply README.md]: corrected on new push

File: README.md (hunk 5 above)
Comment (Developer3027) [reply README.md]: corrected on new push

File: README.md (hunk 6 above)
Comment (Developer3027) [reply README.md]: corrected on new push

File: README.md (hunk 7 above)
Comment (Developer3027) [reply README.md]: corrected on new push

**Pull request merged**